db/migrate/20170103085726_create_cashboxes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateCashboxes < ActiveRecord::Migration
+  def change
+    create_table :cashboxes do |t|
+      t.integer :balance, default: 0
```

Comment:
As per Slack, I think storing `balance` on `Transaction`/`Payment` rather than `Cashbox` is a more robust solution